dist/samples/places-autocomplete-addressform/index.html

```diff
@@ -144,36 +171,34 @@
         return arr2;
       }
 
-      // This sample uses the Autocomplete widget to help the user select a
+      // This sample uses the Places Autocomplete widget to help the user select a
       // place, then it retrieves the address components associated with that
       // place, and then it populates the form fields with those details.
       // This sample requires the Places library. Include the libraries=places
       // parameter when you first load the API. For example:
       // <script
       // src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBIwzALxUPNbatRBj3Xi1Uhp0fFzwWNBkE&libraries=places">
-      var placeSearch;
       var autocomplete;
-      var componentForm = {
-        street_number: "short_name",
-        route: "long_name",
-        locality: "long_name",
-        administrative_area_level_1: "short_name",
-        country: "long_name",
-        postal_code: "short_name",
-      };
+      var address1Field;
+      var address2Field;
+      var stateField;
+      var postalField;
 
       function initAutocomplete() {
-        // Create the autocomplete object, restricting the search predictions to
-        // geographical location types.
-        autocomplete = new google.maps.places.Autocomplete(
-          document.getElementById("autocomplete"),
-          {
-            types: ["geocode"],
-          }
-        ); // Avoid paying for data that you don't need by restricting the set of
-        // place fields that are returned to just the address components.
+        address1Field = document.querySelector("#shipaddress");
```

Comment:
```suggestion
        address1Field = document.querySelector("#ship-address");
```

Might need to check the others.

Reply:
done